src/main/java/de/zalando/ep/zalenium/container/kubernetes/KubernetesContainerClient.java

```diff
@@ -577,7 +591,7 @@ public static DoneablePod createDoneablePodDefaultImpl(PodConfiguration config)
                             .withInitialDelaySeconds(5)
                             .withFailureThreshold(60)
                             .withPeriodSeconds(1)
-                            .withTimeoutSeconds(1)
+                            .withTimeoutSeconds(5)
```

Comment:
Is your environment slow enough, that you're never passing a health check?

Reply:
Are you using video recording?

Reply:
How large are your k8s nodes in cpu and memory?

Reply:
Video recording is currently switched off.

Here are the resources for a given node fetched via `kubectl get pod zalenium-40000-<UID>`

```   resources:
      limits:
        cpu: "1"
      requests:
        cpu: 250m
        memory: 4Gi
```

Reply:
Fair enough. 5 seconds is ok, I can live with that default.